### PR TITLE
CI: refactor

### PR DESCRIPTION
**Pull request by bdarcus**
Defines a separate prettier job and runs on both json and yaml files.

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout 
+      uses: actions/checkout@v2
+    - name: Set up npm
+      uses: actions/setup-node@v1
+    - name: Install prettier
+      run: npx prettier@2.0.5 . --write
+    - name: check json and yaml formatting
+      run: npx prettier --check schemas/**/*.{json,yaml}
```
Comment (bdarcus) [inline .github/workflows/test.yaml:16]: @dhimmel - github is complaining about a yaml syntax error here, but I've tried everything I can think of, and nothing works. Any ideas?

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -1,8 +1,19 @@
-name: Python Test Suite
+name: CSL CI Test Suite
 on:
 - push
 - pull_request
 jobs:
+    name: prettier
```
Comment (dhimmel) [inline .github/workflows/test.yaml:6]: you are missing a key

```suggestion
jobs:
  prettier:
    name: prettier
```

File: .github/workflows/test.yaml (hunk 1 above)
Comment (dhimmel) [reply .github/workflows/test.yaml:16]: see https://github.com/citation-style-language/schema/pull/264/files#r442869965

Comment (bdarcus) [reply .github/workflows/test.yaml:16]: OMG; what am embarrassing oversight!

Thanks.

File: .github/workflows/test.yaml [hunk 3]
```diff
@@ -1,8 +1,20 @@
...
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout 
+      uses: actions/checkout@v2
+    - name: Set up npm
+      uses: actions/setup-node@v1
+    - name: Install prettier
+      run: npx prettier@2.0.5 . --write
+    - name: check json and yaml formatting
+      run: npx prettier --check schemas/**/*.{json,yaml}
```
Comment (bdarcus) [inline .github/workflows/test.yaml:17]: this currently fails because prettier complains about missing yaml files

**Pull request merged**